pkg/testutil/images_linux.go

```diff
@@ -0,0 +1,54 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package testutil
+
+import (
+	_ "embed"
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+)
+
+//go:embed images.yaml
+var rawImagesList string
+
+type TestImage struct {
+	Ref      string   `yaml:"ref"`
+	Tag      string   `yaml:"tag,omitempty"`
+	Digest   string   `yaml:"digest,omitempty"`
+	Variants []string `yaml:"variants,omitempty"`
+}
+
+var testImages map[string]TestImage
+
+func getImage(key string) string {
+	var im TestImage
+	var ok bool
+
+	err := yaml.Unmarshal([]byte(rawImagesList), &testImages)
```

Comment:
Cache parsed YAML data to avoid repeated parsing:

```suggestion
    var testImagesOnce sync.Once
    testImagesOnce.Do(func() {
        testImages = make(map[string]TestImage)
        testImagesErr = yaml.Unmarshal([]byte(rawImagesList), &testImages)
    })
```

Reply:
Yep, will do.

Reply:
Done. Thanks!